models/job.test.js

```diff
@@ -0,0 +1,255 @@
+const db = require("../db.js");
+const { BadRequestError, NotFoundError } = require("../expressError");
+const Job = require("./job.js");
+const {
+    commonBeforeAll,
+    commonBeforeEach,
+    commonAfterEach,
+    commonAfterAll,
+} = require("./_testCommon");
+
+beforeAll(commonBeforeAll);
+beforeEach(commonBeforeEach);
+afterEach(commonAfterEach);
+afterAll(commonAfterAll);
+
+/************************************** create */
+
+describe("create", function () {
+    const newJob = {
+        title: "NJ1",
+        salary: 50000,
+        equity: 0.45,
+        companyHandle: "c3"
+    };
+
+    test("works", async function () {
+        let job = await Job.create(newJob);
+        expect(job).toEqual({
+            ...newJob,
+            equity: "0.45",
+            id: expect.any(Number)
+        });
+
+        const result = await db.query(
+            `SELECT id, title, salary, equity, company_handle AS "companyHandle"
+             FROM jobs
+             WHERE title = 'NJ1'`);
+        expect(result.rows).toEqual([
+            {
+                id: expect.any(Number),
+                title: "NJ1",
```

Comment:
Could probably use the same `newJob` object here as well, to minimize risk of errors due to copy-paste.